lib/util.js

```diff
@@ -300,12 +298,9 @@ function stylizeNoColor(str, styleType) {
 
 function arrayToHash(array) {
```

Comment:
I tried that and it was actually a bit slower in my benchmarks. When 6.1 lands I might have another look at it.